.github/actions/spelling/allow/ubuntu-misc.txt

```diff
@@ -0,0 +1,20 @@
+backport
+backports
+ffe
+fixcommitted
+fixreleased
+ftbfs
+inprogress
+lto
+main
+mir
+motu
+motus
+multiverse
+nbs
+restricted
+sru
+triaged
+uife
```

Comment:
should this be "uefi"?

Reply:
Resolved with https://github.com/canonical/ubuntu-mir/pull/19/commits/3e7dd8342c82d456296ba8538f2192b63f6df9ee

Reply:
Thanks for catching this!

Reply:
Ohhh... I remember again that this was the acronym UI Freeze Exception 🤦🏻